code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -72,7 +72,7 @@
 				display_as_wanderer = TRUE
 		else if(job)
 			var/datum/job/J = SSjob.GetJob(job)
-			if(J.wanderer_examine)
+			if(!J || J.wanderer_examine)
```

Comment:
🤔

Reply:
*Shrugs*, Don't look at me, I have no idea what that does.

Reply:
It's presumably a null check that's done in questionable manner. Let's just not, I'll remove this line

Reply:
Actually nevermind, this works I suppose. Should fail more loudly, however, in my opinion.

Reply:
This seemed to runtime for bandits, who didn't have actual jobs.